pages/TheExplorationEditor.html

```diff
@@ -38,8 +38,7 @@
   </h3>
 
   <p>
-    A card consists of three parts: some 'content', an 'interaction' (or
-    question), and 'responses' that specify what Oppia should do next, based on
+    A card consists of three parts: some <strong>content</strong>, an <strong>interaction</strong> (or <strong>question</strong>), and <strong>responses</strong> that specify what Oppia should do next, based on
```

Comment:
Sorry, I missed this: 'question' is not a term of art, so we probably don't need to make it bold.